es-vue-base/src/lib-components/EsFormMsg.vue

```diff
@@ -1,19 +1,21 @@
 <template>
     <b-alert
-        v-if="message"
-        :show="dismissCountDown"
+        :show="dismissCountDown "
         :variant="variant"
         dismissible
         class="form-msg"
         fade
-        @dismissed="dismissCountDown=0"
+        @dismissed="dismissCountDown = 0"
         @dismiss-count-down="countDownChanged">
         <div class="d-flex">
             <div class="pr-50">
                 <IconCircleAlert v-if="variant === 'danger'" />
                 <IconCircleCheck v-if="variant === 'success'" />
+                <IconInfo v-if="variant === 'primary'" />
             </div>
-            {{ message }}
+            <slot>
+                {{ formMsg }}
+            </slot>
```

Comment:
This should be able to just be `<slot />` with nothing inside. And any message can just be passed into the slot when this component is used.